### ISSUE TITLE
Backup in 4.1 and 4.0

### ISSUE THREAD

**Issue by arwinj** [mkucej/i-librarian#46]
Hi.
The backup utility does not work as expected in 4.1 and 4.0. The web interface does not show any errors, and I checked that the backup folder is readable/writable.
Here is an extract of the apache error log :
```
cp: cannot create regular file '/home/Backup/pdfs/01/00338.pdf': No such file or directory
...
cp: cannot create regular file '/home/Backup/database/history.sq3': No such file or directory
...
cp: cannot create regular file '/home/Backup/supplement/._00087graphical_abstract.pdf': No such file or directory
...
```
The library folder is not created like the former versions of i-librarian. Therefore, the backup function does not find the subfolder. Creating them manually allow the backup to finish without error in the log.

Thanks.

Version : 4.0 and 4.1
OS : Linux 3.16.0-38-generic, Ubuntu 14.04.

**Comment by mkucej** [comment]
You need to create writable `library` folder now. There should be some about that hints in the GUI.

**Comment by mkucej** [comment]
Fixed in 4.2.
